tests/test_catalog.py

```diff
@@ -0,0 +1,16 @@
+from scivision.catalog import PandasCatalog
+
+
+class TestPandasCatalogInit:
+    """Checks that PandasCatalogs can be constructed from several argument
+    types (currently just that no exceptions are raised)."""
+
+    def test_init_default(self):
+        """Load the default catalog"""
+        PandasCatalog()
+
+    def test_init_filenames(self):
+        """Load catalog from explicit paths"""
+        PandasCatalog(
+            "tests/test_datasource_catalog.json", "tests/test_model_catalog.json"
+        )
```

Comment:
I would suggest to add tests for the `_corerce` and `_compatible` functions

Reply:
Enhanced these a bit (I left out _coerce*, since they're 'private' and covered by the constructor).